python/ml4ir/base/config/keys.py

```diff
@@ -98,3 +98,12 @@ class EncodingTypeKey(Key):
 
     BILSTM = "bilstm"
     CNN = "cnn"
+
+
+class DefaultDirectoryKey(Key):
+    """Default directory paths"""
+
+    MODELS = "models"
+    LOGS = "logs"
+    DATA = "data"
+    TEMP_DATA = "data/temp"
```

Comment:
since they are temp, should we move this to `/tmp/data`? do we have cleaning in place?

Reply:
although I don't know if flowsnake has /tmp

Reply:
I explicitly delete the temp directory once done.